### ISSUE TITLE
Apply WCAG standards to log in screen

### ISSUE THREAD

**Issue by SLDonnelly** [bcgov/nr-spar#1722]
**As a** *SPAR User* **I want** *SPAR to follow WCAG standards* **So That** *I can use assistive technologies to help me complete my work*

**Acceptance Criteria**

For the SPAR log in screen 
![image](https://github.com/user-attachments/assets/acea99c9-79f4-4482-b302-a49a8520e5a1)

- [x] Semantic HTML structure
- [x] Ensure all images have alternative text (for the photo on the log in screen use: "Picture of spruce tree branches with needles and cones".
- [ ] Accessible forms
- [x] Keyboard navigation
- [x] ARIA (Accessible Rich Internet Applications) roles and attributes
- [ ] Focus management
- [x] Check that contrast is above 4.5:1
- [x] Tested - Use accessibility tools and screen readers during development.
   
**Additional Context**
- More information here: https://github.com/bcgov/nr-spar/wiki/Coding-for-accessibility:-Screen-readers

**Definition of Done**
- [x] Ready to Demo in Sprint Review
- [x] Does what I have made have appropriate test coverage?
- [x] Documentation and/or scientific documentation exists and can be found
- [x] Peer Reviewed by 2 people on the team
- [ ] Manual testing of all PRs in Dev and Prod
- [ ] QA from UX team member
- [ ] Merged

**Comment by ngunner15** [comment]
To Do before working on WCAG tickets:

- Read up on the documentation for various HTML element's WCAG requirements.
- Example scenario
> What is the correct tag for a text? Is it `<h1>`, `<p>` or for styling purposes it should be `<div>`. If we use `<div>` and it should behave as a heading or title then we have to add aria labels to show `<div>` as a heading.
eg `<div role="heading" aria-level="2">Title Text Here</div>`

- Find a screen reader testing tool preferably one that can be integrated with a browser like a Chrome extension, for eg [WAVE](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)

**Comment by kevinginley** [comment]
@ngunner15 I did a manual review using WAVE and the page is meeting the outlined criteria. Great work!
